packages/flutter_tools/schema/material_fonts.yaml

```diff
@@ -1,4 +0,0 @@
-material:
```

Comment:
No reason to load this from a file: its unchanged for years, and really the dart code is easier to update

Reply:
well, changed once in 5 years